src/main/scala/cz/alenkacz/db/postgresscala/Connection.scala

```diff
@@ -0,0 +1,13 @@
+package cz.alenkacz.db.postgresscala
+
+import scala.concurrent.Future
+
+trait Connection {
+    def isConnected: Boolean
+
+    def sendQuery[T](query: String, deserializer: Row => T): Future[Seq[T]]
+    def sendQuery[T](query: String): Future[Option[T]]
```

Comment:
If I understand the implementation correctly then this overload should be used for query that returns one row with one value, right? If so then I would prefer to choose better name, e.g. `queryValue`.